api/controllers/main.js

```diff
@@ -87,7 +71,7 @@ module.exports = {
         title: 'Unauthorized',
         message: 'You are not permitted to perform this action. Are you sure you are logged in?',
       });
-      return res.redirect('/');
+      return res.redirect(config.app.homepageUrl);
```

Comment:
We set a flash message above this, but that isn't going to have any effect if we redirect to the static site.

Reply:
Good catch. Fixed in most recent version.